pages/_app.tsx

```diff
@@ -27,6 +28,7 @@ export default function App({ Component, pageProps: {session, ...pageProps} }: A
         <NationDialog>
           <MainNav />
           <WebThreeDialog />
+          <WrongNetworkModal requiredNetwork="Kovan" />
```

Comment:
wrong-network-modal HAS to be used by network-identifier

Reply:
Finish